### ISSUE TITLE
phalanx fracture

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10703]
**Preferred term label:**
phalanx fracture

**Synonyms**

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Phalanx fracture
Seymour fracture
Mallet finger

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
It would be a little tidier if we made this Finger phalanx fracture, which seems to match with the diseases listed above. Is that OK on your side?
